connection.go

```diff
@@ -293,6 +300,17 @@ type SslOpts struct {
 	Ciphers string
 }
 
+// Clone returns a copy of the Opts object.
+// Beware that Notify channel, Logger and Handle are not copied.
```

Comment:
It seems confusing:
```suggestion
// Clone returns a copy of the Opts object.
// Beware that Notify channel, Logger and Handle are not copied.
```
because they are copied in fact:
```
opts.Close().Notify == opts.Notify
```

Reply:
Removed `// Beware that Notify channel, Logger and Handle are not copied.` line